torchrl/data/replay_buffers/storages.py

```diff
@@ -146,10 +146,14 @@ def _empty(self):
 
     def _rand_given_ndim(self, batch_size):
         # a method to return random indices given the storage ndim
+        if isinstance(self, TensorStorage):
```

Comment:
This is almost definitely too hacky, but I just wanted to check if this would fix the failures